src/application/node/application.ts

```diff
@@ -23,6 +24,7 @@ export interface BackendApplicationContribution {
 export class BackendApplication {
 
     private app: express.Application;
+    @inject(ILogger) private logger: ILogger;
```

Comment:
Yes I just wanted such a helper to be out of the "core" depencies, I though a property was a good do that..
Other opinions on this?